profile.d/toolbox.sh

```diff
@@ -27,7 +27,7 @@ if [ -f /run/ostree-booted ] \
    && { [ "${VARIANT_ID}" = "workstation" ] || [ "${VARIANT_ID}" = "silverblue" ] || [ "${VARIANT_ID}" = "kinoite" ]; }; then
     echo ""
     # shellcheck disable=SC3059
-    echo "Welcome to Fedora ${VARIANT_ID^}. This terminal is running on the"
+    echo "Welcome to Fedora ${VARIANT:-$VARIANT_ID}. This terminal is running on the"
```

Comment:
Unfortunately, both `VARIANT` and `VARIANT_ID` are [optional](https://www.freedesktop.org/software/systemd/man/os-release.html) fields.  Hence we can't rely on either of them.

Reply:
I think, these days we should use `PRETTY_NAME` and let the downstream distributor decide how the host operating system should be called. `PRETTY_NAME` has the advantage of being effectively a mandatory field. The os-release(5) [manual](https://www.freedesktop.org/software/systemd/man/os-release.html) says:
```
If not set, a default of "PRETTY_NAME="Linux"" may be used.
```

Note that the [original](https://github.com/containers/toolbox/commit/79f59b667b22b647253cb047a9c04cd60d1db03a) welcome banner was from a time when on Fedora Silverblue `VARIANT` and `VARIANT_ID` were set to the same values as on Fedora Workstation. That's why it started off hard-coded as Fedora Silverblue.

Fedora Silverblue [got its own](https://pagure.io/workstation-ostree-config/c/c18ef957d11862d32f362722931dbfdf1f5beb0d) `VARIANT` and `VARIANT_ID` only as recently as Fedora 32.

Reply:
So, this needs to be changed to:
```
echo "Welcome to Fedora ${PRETTY_NAME:-Linux}."
```

Since, `PRETTY_NAME` tends to be quite long, and is out of our control, it will be good to break the next sentence into a separate paragraph.  That will ensure that the banner isn't too wide and fits on a wide variety of terminals.  Something like:
```
echo ""
echo "This terminal is running ..."
```

Reply:
Sorry, just `${PRETTY_NAME:-Linux}`, not `Fedora ${PRETTY_NAME:-Linux}`.

Reply:
Done in https://github.com/containers/toolbox/pull/1177